kfs-core/src/main/java/edu/arizona/kfs/gl/service/impl/ScrubberValidatorImpl.java

```diff
@@ -219,16 +220,15 @@ protected Message validateGTEAccount(OriginEntryFull originEntry) {
         account.refreshReferenceObject("subFundGroup");
```

Comment:
Quick question: does the mere validation of the account, financialObject and subFundGroup are worth 3 separate trips to the DB to refresh the objects first?